assets/js/modules/thank-with-google/components/settings/SettingsView.stories.js

```diff
@@ -52,28 +70,56 @@ function Template() {
 export const Default = Template.bind( null );
 Default.storyName = 'Default';
 Default.parameters = { features };
+Default.args = {
+	setupRegistry: ( registry ) => {
+		registry.dispatch( CORE_SITE ).receiveSiteInfo( {
+			postTypes: [
+				{ slug: 'posts', label: 'Posts' },
+				{ slug: 'pages', label: 'Pages' },
+				{ slug: 'custom', label: 'Custom' },
+			],
+			widgetsAdminURL: 'http://example.com/wp-admin/widgets.php',
+		} );
+		registry.dispatch( MODULES_THANK_WITH_GOOGLE ).receiveGetSettings( {
+			publicationID: 'example.com',
+			buttonPlacement: 'static_auto',
+			colorTheme: 'purple',
+			buttonPostTypes: [ 'posts', 'pages' ],
+		} );
+	},
+};
+
+export const SettingsError = Template.bind( null );
```

Comment:
For some reason this story doesn't display the post types value: https://google.github.io/site-kit-wp/storybook/pull/5603/?path=/story/modules-thank-with-google-settings-settingsview--settings-error. Could you please take a look why?